coursedata/adventures/nl.yaml

```diff
@@ -1547,6 +1620,29 @@ adventures:
                         print ' Hoera! Je bent ontsnapt!'
                     ```
                 start_code: "print 'Ontsnap uit het spookhuis!'"
+    piggybank:
+            name: Spaarvarken
+            description: Tel je zakgeld!
+            image: hauntedhouse.png
```

Comment:
Hebben we hier nog een Felienne-original nodig? Maak maar een issue zodat ik het niet vergeet :)

Reply:
Check!